pkg/phases/konfigadm.go

```diff
@@ -56,10 +56,21 @@ func CreatePrimaryMaster(platform *platform.Platform) (*konfigadm.Config, error)
 
 func baseKonfig(platform *platform.Platform) (*konfigadm.Config, error) {
 	platform.Init()
-	cfg, err := konfigadm.NewConfig().Build()
-	if err != nil {
-		return nil, fmt.Errorf("baseKonfig: failed to get config: %v", err)
+	var cfg *konfigadm.Config
+	var err error
+
+	if platform.Master.KonfigadmFile == "" {
```

Comment:
This would use the master konfigadm file for all VM's? If I specify a konfigadm file under master, I wouldn't expect it to be users under workers and vice versa. You may also want to use different konfigadm for different worker pools

Reply:
:thinking: , yes.

Cool. Will rework this into being specific per master or worker pool.